types/ember/test/component.ts

```diff
@@ -12,3 +13,100 @@ Component.extend({
 Component.extend({
     layout: 'my-layout'
 });
+
+const component1 = Component.extend({
+  actions: {
+    hello(name: string) {
+      console.log('Hello', name);
```

Comment:
It would be nice to test that the context gets set properly, e.g. by calling `this.get('someComponentProperty')`